Example/SmileID/Home/HomeView.swift

```diff
@@ -53,6 +53,40 @@ struct HomeView: View {
                                 )
                             }
                         ),
+                        ProductCell(
```

Comment:
Yeah, I plan to comment them out before merging to main. For now I added it so that it's easier for anyone to pull the branch and test if they need to. Similar to how it is in the Android codebase - https://github.com/smileidentity/android/blob/69c8d7d66ac6db9f53eb4d62bf9e610b146730f2/sample/src/main/java/com/smileidentity/sample/compose/MainScreen.kt#L215